complete_streaming_function.py

```
@@ -0,0 +1,394 @@
+                # Store current scene ID for next comparison
+                orchestrator._last_scene_id = current_scene_id
+            
+            current_scene = orchestrator.scenario.get('scenes', [{}])[orchestrator.state.current_scene_index]
+            timeout_turns = current_scene.get('timeout_turns') or current_scene.get('max_turns', 15)
+            correct_scene_id = current_scene.get('id')
+            
+            # Handle "begin" command
+            if request.message.lower().strip() == "begin":
+                # Check if simulation is already started - if so, ignore the begin command
+                if orchestrator.state.simulation_started:
+                    print(f"[STREAM DEBUG] Simulation already started, ignoring 'begin' command")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)